taskcoordinator/src/main/java/com/microservices/taskcoordinator/service/OrderService.java

```diff
@@ -0,0 +1,15 @@
+package com.microservices.taskcoordinator.service;
+
+import com.microservices.taskcoordinator.dto.OrderDTO;
+import com.microservices.taskcoordinator.dto.inbound.OrderCoordinationDTO;
+import com.microservices.taskcoordinator.dto.outbound.OrderSubmissionDTO;
+
+public interface OrderService {
+
+    OrderDTO updateOrder(OrderDTO order);
+
+    OrderDTO getOrderById(Integer id);
```

Comment:
why Integer not int?

Reply:
Ok, that might be

Reply:
? I guess that it has to be int not Integer. Why do u think Integer is ok?